components/block-description/index.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
I guess this component should be moved to the `blocks` module next to the `BlockControls`, `InspectorControls`, ...